### PR TITLE
Add more liter-based volume rates.

### PR DESCRIPTION
**Pull request by Aehmlo**
I totally forgot to do this in #116, but it was the main motivation for doing that (d'oh!).

### CODE REVIEW

File: src/si/volume_rate.rs [hunk 1]
```diff
@@ -56,6 +56,28 @@ quantity! {
         @cubic_yoctometer_per_second: prefix!(yocto) * prefix!(yocto) * prefix!(yocto);
             "ym³/s", "cubic yoctometer per second", "cubic yoctometers per second";
 
+        @yottaliter_per_second: prefix!(milli) * prefix!(yotta); "YL/s", "yottaliter per second", "yottaliters per second";
```
Comment (iliekturtles) [inline src/si/volume_rate.rs]: Wrap at 100 characters. Just before the plural description.

Comment (Aehmlo) [reply src/si/volume_rate.rs]: Weird that `rustfmt` doesn't catch these.

**Comment by Aehmlo** [comment]
If you're willing, a patch release would be great.

File: src/si/volume_rate.rs (hunk 1 above)
Comment (Aehmlo) [reply src/si/volume_rate.rs]: Ah, from the [list of limitations](https://github.com/rust-lang/rustfmt#limitations) in the `rustfmt` README:

> Macro declarations and uses (current status: some macro declarations and uses are formatted).

**Comment by iliekturtles** [comment]
Thanks! I'll try to get the changes released later today or tonight.

**Comment by iliekturtles** [comment]
[v0.22.1](https://crates.io/crates/uom) released!

**Pull request merged**